packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -25,6 +45,32 @@ declare global {
        * @category Containers util
        */
       startWildflyContainer(): Chainable<unknown>;
+      /**
+       * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       * @category Containers util
+       *
+       * @param configuration - the name of XML configuration file.
+       */
+      startWildflyContainerSecured(): Chainable<unknown>;
+      /**
+       * Start a SSO container. This command which needs to executed in before method to start and configure SSO.
+       *
+       * The network mode must be host. Because to SSO is accessed from a network (Wildfly to Keycloak the IP must be used)
+       * and from outside of a network (The users browser is outside of network and does't know the IP of the SSO).
+       * In configuration of OICD can be used localhost or IP but then users browser or Wildfly cannot access to SSO
+       * Solution is run both service in host mode to be available at the host machine and then can be configured the
+       * OIDC to localhost.
+       *
+       * The configuration file is in /berg/testsuite/cypress/fixtures/realm-configuration.json
+       * How to update the configuration? get generated file with users and roles!
+       * 1) import existing file into standalone keycloak
+       * 2) update configuration in Keycloak and stop server
+       * 3) export new configuration by following command:
+       * ./bin/kc.sh export --realm wildfly-infra --users realm_file --file /path/to/your/file.json
+       *
+       * @category Containers util
+       */
+      startSsoContainer(): Chainable<unknown>;
```

Comment:
I suggest to keep the upstream project names without generalization, i.e. `startSsoContainer` -> `startKeycloakContainer`.

Reply:
new name used `startKeycloakContainer`